src/pnpq/utils.py

```diff
@@ -1,15 +0,0 @@
-from serial.tools.list_ports import comports as list_comports
-import logging
-
-logger = logging.getLogger("utils")
-
-
-def get_available_port(device_serial_number: str) -> str | None:
-    logger.debug(f"get_available_port(serial_number: {device_serial_number})")
```

Comment:
We have almost used get_available_port() for connecting to a specific device regardless of how many devices are connected to the host controller. After removing this file 'utils.py', I could not find the alternative of get_available_port().

Reply:
Please see the PR summary.

> pnpq.utils has been deleted. It duplicated the functionality in pnpq.devices.utils and appears to be unused.